recipes-config/systemd-resource-managament/systemd-resource-managament.bb

```diff
@@ -0,0 +1,84 @@
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=3e927855e47b22eecd81954b0f545a0f"
+
+SRC_URI += " \
+    file://LICENSE \
+    file://audio.slice \
+    file://connectivity.slice \
+    file://coreservices.slice \
+    file://dlt.slice \
+    file://media.slice \
+    file://phone.slice \
+    file://speech.slice \
+    file://swdl.slice \
+    file://uiapps.slice \
+    file://ui.slice \
+    file://vehiclefunctions.slice \
+    file://video.slice \
+"
+
+S = "${WORKDIR}"
+
+inherit allarch
+
+do_install_append() {
+    install -d ${D}${systemd_unitdir}/system/
+
+    # Slices
+    install -d ${D}${systemd_unitdir}/system/audio.slice.d/
+    install -m 0644 ${WORKDIR}/audio.slice ${D}${systemd_unitdir}/system/audio.conf
+
+    install -d ${D}${systemd_unitdir}/system/connectivity.slice.d/
+    install -m 0644 ${WORKDIR}/connectivity.slice ${D}${systemd_unitdir}/system/connectivity.conf
```

Comment:
typo, destination file name prefix shall be slice, not conf.

But, the question is however if we shall create a slice configuration file (slice.conf) and put it as an override into the <slice name>.slice.d/ folder instead of creating the slice unit file in the /system folder.

See other comment

Reply:
I'm thinking of 2 scenarios:
1. Create a new slice, like connectivity. In this case, one may create a new slice unit file, like connectivity.slice, and all slice configurations are put here and finally installed into /lib/systemd/system/<slice name>.slice.
2. Adapt an existing slice, like system.slice. In this case, one may create a override by adding slice.system.conf to the layer which is then installed into the /lib/systemd/system/system.slice.d/slice.conf.

For the services, it's all about override becase it's assumed that the service exists. So in this case, the configuration file installed into /lib/systemd/system/<service name>.service.d/service.conf.